### ISSUE TITLE
[BUG] Game suddenly deletes config.xml and UsersData.xml - cannot restore

### ISSUE THREAD

**Issue by Otter7777** [Stephan-S/FS19_AutoDrive#1456]
**AutoDrive Version:** 1.1.0.4   
**Release or Custom Zip:** Release  
**Map (If applicable):** Felsbrunn
**Vehicle / Type (If applicable):**  N/A
**Multiplayer or Singleplayer:** Singleplayer  

**Describe the bug**  
First I noticed that when I added any new mod to my career game that my AutoDrive_Felsbrunn_config.xml and AutoDriveUsersData.xml files were suddenly deleted from my save game.  Then I stopped adding new Mods and played. It went well for awhile then suddenly today it randomly deleted them again.

Nothing in particular happens in the game play to trigger it (at least that I know). My save game from 17:33 has them in it.  All save games after don't.

All other files including careersavegame are still there and the game plays normally just not autodrive.  

Any attempts to copy and paste those files from previous savegameBackups (similar to what you do if the careerSavegame.xml file suddenly deletes don't work.  When you return to the game the Autodrive is back to it's default routes (just like you added it as a new mod).

Therefore you lose all the routes you created on your farm and start at zero again...

System: Using macOS Mojave 10.14.6 on MacBookPro 15 inch 2017
FS19 crashes occasionally for no reason

**If applicable - Steps to reproduce the behavior**  
1. Adding new mods to an existing career save game
or 2. completely random

**Screenshots**  
<img width="632" alt="Autosave 1 with AD files" src="https://user-images.githubusercontent.com/66087461/83182672-fe918400-a126-11ea-818b-9579c8957bcf.png">

<img width="603" alt="Autosave 2 with AS files suddenly gone" src="https://user-images.githubusercontent.com/66087461/83182716-12d58100-a127-11ea-806c-415b48a0862c.png">

**Comment by Otter7777** [comment]
**Trigger** - Landscaping kills the save file

**Comment by Axel32019** [comment]
Does this mean it is not a special bug of AD?

**Comment by Otter7777** [comment]
I would say that it precisely specific to AD!

** How it happens ** I start with the default AD routes.  I customise my farm and fields with AD routes.  My savegame folder has AD config and AD usersdata files above the FScarreersave.  Then I landscape around my farm.  The FScareersave is unaffected.  The AD files are gone.  So this means that my landscaping somehow deletes the AD files specifically.

**Comment by Axel32019** [comment]
Sorry to bother you again! Will the same happen with Latest Release Candidate: 1.1.0.5-RC3?
Here we removed the default Felsbrunn AD network as it is very old and may disturb with actual AD, including 1.1.04. https://github.com/Stephan-S/FS19_AutoDrive/wiki/Release-Notes

**Comment by Otter7777** [comment]
It's really not a bother! We have to identify bugs and fix them ;)

I downloaded the zip from the RC3 but when I open the game again it comes up as 1.1.0.4 even after it updates.  Am I doing it wrong?

**Comment by Otter7777** [comment]
> It's really not a bother! We have to identify bugs and fix them ;)
> 
> I downloaded the zip from the RC3 but when I open the game again it comes up as 1.1.0.4 even after it updates. Am I doing it wrong?

Never mind.  If I download 1.1.0.5 and replace the folder DO NOT update it in the mod section otherwise it reverts you back to 1.1.0.4

I will test things out with the new version and let you know what happens

**Comment by Otter7777** [comment]
> Sorry to bother you again! Will the same happen with Latest Release Candidate: 1.1.0.5-RC3?
> Here we removed the default Felsbrunn AD network as it is very old and may disturb with actual AD, including 1.1.04. https://github.com/Stephan-S/FS19_AutoDrive/wiki/Release-Notes

Axel32019 I wanted to ask if there is a basic map with the road network and buy/sell points?  Or do we have to start from zero?  Thanks!

**Comment by Axel32019** [comment]
Did you had a look at this: https://www.modhoster.com/mods/autodrive-kurse-fur-felsbrunn-by-mc ?
Your issue with MacOS is not known to us up to now and I am not sure if it really depends on AD.

**Comment by Otter7777** [comment]
As for the OS problem I have spent the last days downgrading to Mojave again as I had problems with several apps.  At least I was able to identify a potential problem that other Mac users might encounter.  

Another problem I experienced could be related:

https://forum.giants-software.com/viewtopic.php?f=964&t=167833

I'm not sure if this is related directly to AD but it seems to affect only the AD save and config files.  Naturally that is why I identified it to you.

I have reloaded OS Mojave and AD 1.1.0.5 fresh and will continue to test this out on a complete new game.

Thanks for the link regarding the course.  Yes, I have tried the mod today but the read me text isn't translating so clearly to english because the read me text is written somewhat in German 'shorthand' so I will continue to try to figure out the installation.

Thank you for your advice with this issue and I also hope that identifying it to you was helpful.
